layouts/partials/dbm/dbm-setup-postgres.html

```diff
@@ -23,6 +23,11 @@
           {{ partial "img.html" (dict "root" . "src" "integrations_logos/google_cloudsql.png" "class" "img-fluid" "alt" "Google Cloud SQL" "width" "175") }}
         </div>
       </a>
+      <a class="card" href="/database_monitoring/setup_postgres/azure">
+        <div class="card-body text-center py-2 px-1">
+          {{ partial "img.html" (dict "root" . "src" "integrations_logos/azure_db_for_postgresql.png" "class" "img-fluid" "alt" "PostgreSQL" "width" "175") }}
+        </div>
```

Comment:
![Screen Shot 2022-04-01 at 2 34 08 PM](https://user-images.githubusercontent.com/14317240/161344179-82437cdd-a3a5-4fd8-8df2-1385ca728b33.png)
 
 I added this to the `Setting up Postgres` page, but Azure is not showing up a link in the side panel. Not sure where to add it?

Reply:
The sidenav items get added manually in menus.yaml files, usually we talk about this when you're planning to add new doc pages, and your docs writer takes care of it. I'll add these for you.